kotlin/feature-layer-definition-expression/src/main/java/com/esri/arcgisruntime/sample/featurelayerdefinitionexpression/MainActivity.kt

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2020 Esri
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package com.esri.arcgisruntime.sample.featurelayerdefinitionexpression
+
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import com.esri.arcgisruntime.data.ServiceFeatureTable
+import com.esri.arcgisruntime.geometry.Point
+import com.esri.arcgisruntime.geometry.SpatialReferences
+import com.esri.arcgisruntime.layers.FeatureLayer
+import com.esri.arcgisruntime.mapping.ArcGISMap
+import com.esri.arcgisruntime.mapping.Basemap
+import kotlinx.android.synthetic.main.activity_main.*
+
+class MainActivity : AppCompatActivity() {
+
+  // set flag for applying expression to feature layer
+  private var applyExpression: Boolean = true
+
+  override fun onCreate(savedInstanceState: Bundle?) {
+    super.onCreate(savedInstanceState)
+    setContentView(R.layout.activity_main)
+
+    // create a service feature table from a URL
+    val serviceFeatureTable = ServiceFeatureTable(getString(R.string.sample_service_url))
+    // create a feature layer using the service feature table
+    val featureLayer = FeatureLayer(serviceFeatureTable)
+
+    // create a map with a topographic base map
+    ArcGISMap(Basemap.createTopographic()).let { map ->
+      // add the feature layer to the map
+      map.operationalLayers.add(featureLayer)
+      // set the map to be displayed in the layout's map view
+      mapView.map = map
+    }
+
+    // set a viewpoint on the map view to center on San Francisco
+    mapView.setViewpointCenterAsync(
+      Point(-13630845.0, 4544861.0, SpatialReferences.getWebMercator()), 200000.0
+    )
```

Comment:
```suggestion
    // create a map with a topographic base map
    val map = ArcGISMap(Basemap.createTopographic()).apply {
      // add the feature layer to the map
      operationalLayers.add(featureLayer)
    }

    mapView.apply {
      // set the map to be displayed in the layout's map view
      this.map = map

      // set a viewpoint on the map view to center on San Francisco
      setViewpointCenterAsync(
        Point(-13630845.0, 4544861.0, SpatialReferences.getWebMercator()), 200000.0
      )
    }
```

I think this might be a bit more readable as it sticks with that builder pattern of the map being added to the map view rather than during the building of the map it gets added to the map view